CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributes
+
+To create an extension or run Nango locally: [check our documentation](https://docs.nango.dev/contribute)
+
+## Develop locally
+
+To develop on the platform locally follow those steps:
+
+```sh
+git clone https://github.com/NangoHQ/nango.git
+```
+
+Install the project
+
+```sh
+npm run install:all
+```
+
+Set your envs
+
+```sh
+cp .env.example .env
+```
+
+Launch the services
+
+```sh
+npm run dev:docker
+```
+
+Launch Nango
+
+```sh
+# In two different shell
+npm run dev:watch
+npm run dev:watch:apps
```

Comment:
unrelated to the PR iteselft but I wonder if we could come up with a single command to do that.

Reply:
That would be great in theory but I think it would make the logs a bit unreadable, maybe to test with turborepo (or a bash script)